crypto/evp/evp_err.c

```diff
@@ -23,6 +23,10 @@ static const ERR_STRING_DATA EVP_str_reasons[] = {
     {ERR_PACK(ERR_LIB_EVP, 0, EVP_R_BUFFER_TOO_SMALL), "buffer too small"},
     {ERR_PACK(ERR_LIB_EVP, 0, EVP_R_CAMELLIA_KEY_SETUP_FAILED),
     "camellia key setup failed"},
+    {ERR_PACK(ERR_LIB_EVP, 0, EVP_R_CAN_NOT_GET_PARAMETERS),
+    "can not get parameters"},
```

Comment:
cannot is one word

Reply:
I was waiting for the spelling police :)

Reply:
You got me looking.

https://english.stackexchange.com/questions/132600/can-not-vs-cannot

Will make mental note

Reply:
rebased with updated error names

Reply:
It can be either.....

_I can not get the widget_
_I cannot get the widget_

Both are correct English.